tests/test_pqserver.py

```diff
@@ -198,16 +198,14 @@ def test_flexible_metadata(client):
     assert result["required_keys"] == ["values"]
 
     response = client.post(
-        "/consdb/insert/latiss",
+        "/consdb/insert/latiss/exposure/obs/2024032100003",
```

Comment:
A little weird to have this be different from the values, but perhaps that's an even better test.